addons/mail/models/mail_message.py

```diff
@@ -1038,7 +1049,7 @@ def message_fetch_failed(self):
         return messages._message_notification_format()
 
     @api.model
-    def message_fetch(self, domain, limit=20, moderated_channel_ids=None):
+    def message_fetch(self, domain, fetch_followers=False, limit=20, moderated_channel_ids=None):
```

Comment:
Same reasoning, try to keep the position of existing params to avoid breaking stuff.